apps/common/src/main/scala/org/lfdecentralizedtrust/splice/environment/NodeBase.scala

```diff
@@ -472,4 +361,139 @@ object NodeBase {
       )
 
   }
+
+  def buildHttpClient(
```

Comment:
note: this change makes it harder to see what actually chagned so i will just review the full method